libgalois/include/katana/TypedPropertyGraph.h

```diff
@@ -270,6 +270,11 @@ class TypedPropertyGraphView : public PGView {
       const std::vector<std::string>& edge_properties);
   static Result<TypedPropertyGraphView<PGView, NodeProps, EdgeProps>> Make(
       PropertyGraph* pg);
+  static Result<TypedPropertyGraphView<PGView, NodeProps, EdgeProps>> Make(
+      PropertyGraph* pg, const std::vector<std::string>& node_properties,
+      const std::vector<std::string>& edge_properties,
+      const std::vector<std::string>& node_types,
+      const std::vector<std::string>& edge_types);
```

Comment:
I would much prefer projecting the underlying `PropertyGraph` and then creating a `TypedPropertyGraphView` from the untyped PropertyGraphView. The less TypedPropertyGraph does the better since it's really just a C++ utility class, Python does not use it.

Reply:
Yeah, I have created a new fucntion that takes in the pg_view as the argument, and removed this function.